editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -393,7 +393,7 @@
                     {:text (value->label x)})
      :cell-factory (fn [x]
                      {:text (value->label x)})
-     :items (sort (mapv first options))}))
+     :items (mapv first options)}))
```

Comment:
The enforced sorting of items in ChoiceBox dropdowns resulted in strange orderings for enum value options. I think it should be your responsibility to pre-sort the `options` you provide if you want them sorted.

Not sure we use these dropdowns for a lot of stuff except enums anyway.

Reply:
Nice catch! I'll go over them and selectively sort the options where it makes sense!